### DISCUSSION TITLE
Create console command to create classes for Requests and Responses

### DISCUSSION THREAD

**Discussion by ExFuseMe** [DarkaOnLine/L5-Swagger#627] [Ideas]
Hello, esteemed community. 

In many projects, I often encounter the need to write PHP classes for Requests and Responses, as the code examples provided in `swagger-php` examples do not always fully meet the requirements for solving such tasks. 

Therefore, I would like to discuss the possibility of implementing an Artisan command for generating these classes (similar to how it's implemented in Laravel with commands like `artisan make:controller`, etc.). 

For example, we could introduce a command like:  
`artisan l5-swagger:make-schema SchemaName`

What are your thoughts on this?

**Comment by DarkaOnLine** [comment]
Hi, sounds interesting! 

You can proceed with PR or before that give an example what you think this command will generate so we could have discussion before you put code in

**Reply by ExFuseMe** [reply]
I think this command will generate a new class in app/Swagger folder(without any fields). Also we could add flags --request and --resource to place them in subfolders Requests and Resources. For example, that's what i use to add swagger schema for pet and owner schemas(example like in `php-swagger`)
![image](https://github.com/user-attachments/assets/af6ba3b8-9d8f-4c58-b65d-03adc02bdbc7)
![image](https://github.com/user-attachments/assets/065cbc3c-0fb9-462f-bf4b-ba259fff7fd7)
*P.S. Also i think we could create add helper --table=`DB table name` to generate schemas with table's fields*

**Reply by DarkaOnLine** [reply]
Sounds good. I would use `app/OpenApi` namespace instead of `Swagger` and I thing name space should be configurable with `--namespace` or something so we can use it instead of separate `--request` or `--response` like `--namespace=app/OpenApi/Request`

So maybe let's start with this and see how it goes. After that we can add `--table` as you suggested?

**Reply by ExFuseMe** [reply]
as you wish. But When i forked lib and then run `docker-compose build` i catch:
![image](https://github.com/user-attachments/assets/3880e99a-0838-4e75-86ea-65b0702c0307)

**Reply by ExFuseMe** [reply]
where could be a problem? I use wsl2 with ubuntu
